zebra/rt_netlink.c

```diff
@@ -2532,47 +2532,88 @@ int kernel_neigh_update(int add, int ifindex, uint32_t addr, char *lla,
 				    addr, lla, llalen, ns_id);
 }
 
-/*
- * Add remote VTEP to the flood list for this VxLAN interface (VNI). This
- * is done by adding an FDB entry with a MAC of 00:00:00:00:00:00.
+/**
+ * netlink_update_neigh_ctx_internal() - Common helper api for evpn
+ * neighbor updates using dataplane context object.
+ * @ctx:		Dataplane context
+ * @cmd:		Netlink command (RTM_NEWNEIGH or RTM_DELNEIGH)
+ * @mac:		A neighbor cache link layer address
+ * @ip:			A neighbor cache n/w layer destination address
+ * @hdr_flags:	Extra NLM_* modifiers to NEW request
```

Comment:
Could we change the name of this variable? If these flags are only appended for a NEW request, the variable name should reflect that.

Or better yet, it looks like the only difference is that the flag is REPLACE or APPEND. So maybe just make this a boolean for `replace` and decide whether to append or replace from that.

Reply:
I changed this in the way you proposed.